modules/storage-api/src/main/java/org/apache/ignite/internal/storage/MvPartitionStorage.java

```diff
@@ -268,12 +268,28 @@ interface Locker {
     void updateLease(long leaseStartTime);
 
     /**
-     * Return the start time of the known lease for this replication group.
+     * Returns the start time of the known lease for this replication group.
      *
      * @return Lease start time.
      */
     long leaseStartTime();
 
+    /**
+     * Returns the <em>estimated size</em> of this partition.
+     *
+     * <p>It corresponds to the amount of rows in this partition that satisfies the following restrictions:
```

Comment:
A number of rows, not amount? I know that your English is better, so please correct me if I'm wrong